packages/EasyStandard/readme.md

```diff
@@ -18,6 +18,48 @@ We recommend to use [Composer][1] to manage your dependencies. You can require t
 $ composer require --dev eonx/easy-standard
 ```
 
```

Comment:
I think we need links to the newly added MD files from this one.

Reply:
As i see in other packages, we don't add links here to another MD files. Look at next files for example:
- packages/EasyApiToken/readme.md
- packages/EasyCore/readme.md

Reply:
Ok, let's see how it will look like in the generated docs.